android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -92,7 +95,7 @@ class XMTPModule : Module() {
         Name("XMTP")
         Events("sign", "authed", "conversation", "message")
 
-        Function("address") {
+        Function("address") { clientAddress: String ->
```

Comment:
App wouldn't compile unless I added these but haven't hooked up the testing stuff to actually use them.